Acceptance_test.go

```diff
@@ -18,46 +21,211 @@ import (
-		}
+	BeforeEach(func() {
+		TestServer.Clear()
+		factory := rizo.HTTPResponseFactory(func(w http.ResponseWriter) {
+			w.Header().Add("done", "1")
+			w.WriteHeader(http.StatusOK)
+		})
 
-		output, err := test.ExecuteList("./corcel", list)
-		Expect(err).ToNot(BeNil())
+		TestServer.Use(factory).For(rizo.RequestWithPath("/people"))
+	})
 
-		Expect(string(output)).To(ContainSubstring("Request body file not found: missing-file.json"))
+	AfterEach(func() {
+		TestServer.Clear()
 	})
 
-	It("Outputs a summary to STDOUT", func() {
+	Describe("Core Command Line Usage", func() {
+
+		Describe("Iterations", func() {
+
+			It("Plan Usage", func() {
+
+				var plan = fmt.Sprintf(`---
+name: Some Plan
+iterations: 5
+random: false
+workers: 1
+waitTime: 0s
+duration: 0s
+jobs:
+    - name: Some Job
+      steps:
+      - name: Some Step
+        action:
+          type: HttpRequest
+          httpHeaders:
+            key: 1
+          method: GET
+          url: %s`, TestServer.CreateURL("/people"))
+
+				summary, err := test.ExecutePlanFromData(plan, "--summary")
+
+				Expect(err).To(BeNil())
+
+				Expect(summary.TotalRequests).To(Equal(float64(5)))
+			})
+
+			It("List Usage", func() {
+				list := []string{
+					fmt.Sprintf(`%s -X GET'`, TestServer.CreateURL("/people")),
+				}
+				summary, err := test.ExecuteList(list, "--summary", "--iterations", "5")
+				Expect(err).To(BeNil())
+
+				Expect(summary.TotalRequests).To(Equal(float64(5)))
+			})
+		})
+
+		Describe("Workers", func() {
+
+			It("Plan Usage", func() {
+
+				var plan = fmt.Sprintf(`---
+name: Some Plan
+iterations: 0
+random: false
+workers: 5
+waitTime: 0s
+duration: 0s
+jobs:
+    - name: Some Job
+      steps:
+      - name: Some Step
+        action:
+          type: HttpRequest
+          httpHeaders:
+            key: 1
+          method: GET
+          url: %s`, TestServer.CreateURL("/people"))
+
+				summary, err := test.ExecutePlanFromData(plan, "--summary")
+
+				Expect(err).To(BeNil())
+
+				Expect(summary.TotalRequests).To(Equal(float64(5)))
+			})
+
+			It("List Usage", func() {
+				list := []string{
+					fmt.Sprintf(`%s -X GET'`, TestServer.CreateURL("/people")),
+				}
+				summary, err := test.ExecuteList(list, "--summary", "--workers", "5")
+				Expect(err).To(BeNil())
+
+				Expect(summary.TotalRequests).To(Equal(float64(5)))
+			})
+		})
+
+		Describe("Wait Time", func() {
+
+			It("Plan Usage", func() {
+
+				var plan = fmt.Sprintf(`---
+name: Some Plan
+iterations: 5
+random: false
+workers: 1
+waitTime: 1s
+duration: 0s
+jobs:
+    - name: Some Job
+      steps:
+      - name: Some Step
+        action:
+          type: HttpRequest
+          httpHeaders:
+            key: 1
+          method: GET
+          url: %s`, TestServer.CreateURL("/people"))
+
+				summary, err := test.ExecutePlanFromData(plan, "--summary")
+
+				Expect(err).To(BeNil())
+
+				var runningTimeToParse = strings.Replace(summary.RunningTime, "s", "", -1)
```

Comment:
Seeing the repetition here, I think it might be worth introducing either a function, or better yet a custom matcher (http://onsi.github.io/gomega/#adding-your-own-matchers) so you could just do something like:

```go
Ω.(summary.RunningTime).Should(EqualDurationFromString(5))
```